### ISSUE TITLE
Perhaps document JVM behaviour when running the bytecode?

### ISSUE THREAD

**Issue by half-cambodian-hacker-man** [ItzSomebody/StopDecompilingMyJava#1]
E.g. is the integer autoboxed by the JVM in Entry 6? Does Entry 5 affect reflection?

**Comment by ItzSomebody** [comment]
Yeah, it gets autoboxed (there’s an example in Radon’s normal Indy transformer iirc). An yeah, I believe 006 affects reflection since @ByteCode-Dan reported an issue caused by this on Radon’s repo.
